src/components/SearchBar/SettingsDrawer.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
```

Comment:
note: we will likely changes these icons for some custom ones by the wonderful Fang, but this looks good for now!!